charts/coredns/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: coredns
-version: 1.24.1
+version: 1.24.2
```

Comment:
I don't think so because this is a simple refactor, not a breaking change so a simple path upgrade would be fine. We would increase it to 1.25.x when we produce a breaking change.

@mrueg

Reply:
Well breaking changes would be major bumps, but I agree this is somewhat between a feature (=minor) and some internal change (=patch).